src/components/SlateEditor/plugins/definitionList/index.tsx

```diff
@@ -0,0 +1,174 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import { Descendant, Editor, Element, Path, Transforms, Node } from 'slate';
+import { jsx as slatejsx } from 'slate-hyperscript';
+import { RenderElementProps } from 'slate-react';
+import { colors, fonts } from '@ndla/core';
+import { SlateSerializer } from '../../interfaces';
+import { defaultBlockNormalizer, NormalizerConfig } from '../../utils/defaultNormalizer';
+import { KEY_BACKSPACE, KEY_ENTER } from '../../utils/keys';
+import onBackspace from './handlers/onBackspace';
+import onEnter from './handlers/onEnter';
+import { TYPE_DEFINTION_LIST, TYPE_DEFINTION_DESCRIPTION, TYPE_DEFINTION_TERM } from './types';
+
+export interface DefinitionListElement {
+  type: 'definition-list';
+  children: Descendant[];
+}
+
+export interface DefinitionTermElement {
+  type: 'definition-term';
+  children: Descendant[];
+}
+
+export interface DefinitionDescriptionElement {
+  type: 'definition-description';
+  children: Descendant[];
+}
```

Comment:
Hører disse hjemme i `types`?

Reply:
De kan ligge der, men la de i index for å følge standarden som allerede er her. Helt enig om at de kan ligge types men da burde det gjøres for alle